planning/behavior_path_planner_common/include/behavior_path_planner_common/turn_signal_decider.hpp

```diff
@@ -93,6 +104,11 @@ class TurnSignalDecider
     const TurnSignalInfo & intersection_signal_info, const TurnSignalInfo & behavior_signal_info,
     const double nearest_dist_threshold, const double nearest_yaw_threshold);
 
+  TurnSignalInfo overwrite_turn_signal(
```

Comment:
the function can be `const`

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/7045/commits/1959d3fd77b0a1f2503c8aa3a7eece23dd57f4e9